pyblish_qml/models.py

```diff
@@ -466,6 +466,9 @@ def update_with_result(self, result):
                 item.succeeded = True
                 item.amountPassed += 1
 
+            if item.hasError:
+                item.hasWarning = False
```

Comment:
Ah, does it show yellow even if there's an error? In that case it sounds like an error in the GUI code. Shouldn't need to alter the warning when there's an error.

Having a look..

Aah, yes. If you shuffle [this statement](https://github.com/pyblish/pyblish-qml/blob/7e86c446c8d196527bd7a5d47a0a5f8d5683287b/pyblish_qml/qml/List.qml#L35), to query the error first, then the `hasError` will take prevalence and it won't matter whether it has a warning.

Give it a try.

Reply:
> Aah, yes. If you shuffle this statement, to query the error first, then the hasError will take prevalence and it won't matter whether it has a warning.

Yes ! It did !

Reply:
Wow, much clearer now.

Reply:
it's in fix 5 now.